index.js

```diff
@@ -603,6 +603,11 @@ module.exports = test
 function configure ({ timeout = DEFAULT_TIMEOUT, bail = false, solo = false } = {}) {
   const runner = getRunner()
 
+  if (runner.tests.count > 0 || runner.assertions.count > 0) {
+    throw new Error('ERR_CONFIGURE_FIRST')
```

Comment:
Is this how it is in brittle stable atm? Otherwise I think "Configure before running any tests" is a better error message

Reply:
yep, it's the original brittle error code but they also have their own class `TestError`, so I will use the message type

Reply:
What does the brittle message say?

Reply:
So I use this one "configuration must happen prior to registering any tests"? Without capital letter

```
Brittle: Fatal Error
TestError: configuration must happen prior to registering any tests
    at file:///home/lucas/Desktop/brittle-current/test/fixtures/classic-configure-first.js:9:1
    at ModuleJob.run (node:internal/modules/esm/module_job:198:25)
    at async ESMLoader.import (node:internal/modules/esm/loader:385:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12) {
  code: 'ERR_CONFIGURE_FIRST',
  test: '',
  plan: 0,
  count: 1,
  ended: false
}
```

Reply:
Ya let's use that message - super clear.